### ISSUE TITLE
chunk category repeating across all Element categories

### ISSUE THREAD

**Issue by redplanet** [modxbot/migrate#2880]
redplanet created Redmine issue ID 2880

I have a problem with a chunk category repeating in all other element sections – Templates, TVs, Chunks, Snippets, Plugins, Categories. Is this a bug?

_Check the attached screengrab to see what’s happening_ - the repeating chunk category is named: Mk-1 Chunks (7)
I put a green tick beside the correct/original chunk category which is populated with a sub-cat (head data) and the chunks are inside that. There is a red cross beside the duplicated versions which are all empty – no sub-cat or chunks.

The one difference is in the ‘Categories’ section – this contains the sub-cat ‘head data’ but no chunks.

So to recap, Mk-1 Chunks is fine and populated in Chunks category, but it is duplicated and empty in all other categories, except for ‘Categories’ where there is the sub-cat but no chunks.

One other thing, I’ve tried to move ‘head data’ out of Mk-1 Chunks via click/drag, which works initially, but when I log out and back in ‘head data’ has returned to inside Mk-1 Chunks.

Here’s the server stuff:
Linux x86_64
Apache version 2.2.15
MySQL version 5.0.91-community
PHP version 5.2.13

Browser: generally use up to date FF, but am getting the same in all browsers!!
